src/styles/bloks/stars.css

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("../../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(5) {
+  margin-right: 17px;
+}
```

Comment:
Your `stars` is a block, that means it can be reused, so what if in other case you will need no right margins? Set margins to card__stars element